hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/queries/token/HapiGetTokenInfo.java

```diff
@@ -89,6 +90,17 @@ public HapiGetTokenInfo(String token) {
     private Optional<String> expectedWipeKey = Optional.empty();
     private Optional<String> expectedFeeScheduleKey = Optional.empty();
     private Optional<String> expectedPauseKey = Optional.empty();
+    private boolean emptyPauseKey = false;
+
+    private boolean emptyAdminKeyList = false;
+
+    private boolean emptyFreezeKey = false;
+
+    private boolean emptyKycKey = true;
+
+    private boolean emptySupplyKey = true;
+
+    private boolean emptyWipeKey = true;
```

Comment:
I suppose `emptyKycKey`, `emptySupplyKey` and `emptyWipeKey` has to be also `false` as default values